src/main/kotlin/uk/gov/justice/digital/hmpps/prisonertonomisupdate/activities/ActivitiesResource.kt

```diff
@@ -1,18 +1,57 @@
 package uk.gov.justice.digital.hmpps.prisonertonomisupdate.activities
 
+import com.microsoft.applicationinsights.TelemetryClient
 import io.swagger.v3.oas.annotations.Hidden
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.media.Schema
+import io.swagger.v3.oas.annotations.responses.ApiResponse
 import org.springframework.http.HttpStatus
 import org.springframework.security.access.prepost.PreAuthorize
 import org.springframework.web.bind.annotation.DeleteMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.ResponseStatus
 import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.config.ErrorResponse
+import uk.gov.justice.digital.hmpps.prisonertonomisupdate.config.trackEvent
 import uk.gov.justice.digital.hmpps.prisonertonomisupdate.listeners.EventFeatureSwitch
 
 @RestController
 class ActivitiesResource(
   private val activitiesService: ActivitiesService,
   private val eventFeatureSwitch: EventFeatureSwitch,
+  private val telemetryClient: TelemetryClient,
 ) {
+  @PreAuthorize("hasRole('ROLE_NOMIS_ACTIVITIES')")
+  @PostMapping("/activities/{activityScheduleId}")
```

Comment:
The main change is a new endpoint to emulate receiving an activity created event. 
After problems with synchronisations we fix code/data/config and then end up resyncing - this provides a way to run the sync. again manually.